Paper.tex

```diff
@@ -2510,13 +2510,11 @@ \subsection{Deletion of an Account Despite Out-of-gas}
 
 At block 2675119, in the transaction \seqsplit{0xcf416c536ec1a19ed1fb89e4ec7ffb3cf73aa413b3aa9b77d60e4fd81a4296ba}, an account at address 0x03 was called and an out-of-gas occurred during the call. Against the equation (\ref{eq:pre}), this added 0x03 in the set of touched addresses, and this transaction turned $\boldsymbol{\sigma}[0x03]$ into $\varnothing$.
 
-\section{List of mathematical symbols}\label{app:symbols}
-\begin{tabu*}{X[l]X[c]X} \savetabu{col3}
+\begin{tabu*} spread 0pt{X[l] X[c] X} \savetabu{col3}
```

Comment:
Why is this `spread` not `\spread` or something?  Is this usual?

Reply:
https://www.ctan.org/pkg/tabu
"\be­gin{tabu} spread <di­men> en­larges the en­vi­ron­ment’s “nat­u­ral” width.". spread 0 means that it fits to the natural width. I thought that it looked better this way, rather than spread to the whole page.